src/ansi-c/scanner.l

```diff
@@ -1561,6 +1564,18 @@ __decltype          { if(PARSER.cpp98 &&
                   return TOK_INTEGER;
                 }
 
+{clang_ext_float_s} { if(PARSER.mode!=configt::ansi_ct::flavourt::GCC &&
+                         PARSER.mode != configt::ansi_ct::flavourt::CLANG)
+                      {
+                        yyansi_cerror("Preprocessor directive found");
```

Comment:
The error message should be something else.

Reply:
Fixed (also the one that existed before for GCC).